src/libraries/NetCoreAppLibrary.props

```diff
@@ -105,7 +105,6 @@
       System.ObjectModel;
```

Comment:
I see that you are only removing code from this section. Should the newly added library not be exposed in the shared framework?

Reply:
The new ones are `System.Runtime.InteropServices.JavaScript` and `JSImportGenerator`. 
Both of them should be visible and compile on any platform.
I don't fully understand the question nor the necessary action. Could you please elaborate ?

Reply:
The new assembly System.Runtime.InteropServices.JavaScript is currently not exposed in the shared framework because the entry is missing from here: https://github.com/dotnet/runtime/blob/cf8327316337d7909a00e6207c7335adf1078f04/src/libraries/NetCoreAppLibrary.props#L28-L176.

I see that the source generator is [already exposed](https://github.com/dotnet/runtime/blob/cf8327316337d7909a00e6207c7335adf1078f04/src/libraries/NetCoreAppLibrary.props#L188), so no action necessary for it.

Reply:
It's already there from previous PR
https://github.com/dotnet/runtime/blob/cf8327316337d7909a00e6207c7335adf1078f04/src/libraries/NetCoreAppLibrary.props#L129
is that OK ?